### PR TITLE
DM-35758: Document how to use authList.

### PR DESCRIPTION
**Pull request by tribeiro**
* Move "Announce Component Usage" documentation from "Operational Tutorials" to "Daytime/Nighttime Interactions".
* Remove "Operational Tutorials".
* Update front page to remove content under "System Testing, Deployments, and Upgrades" and add a note that it was moved to obs-controls page.

### CODE REVIEW

**Review by Slimleashma — APPROVED** [review]
I'm down with the new arrangement, including the announcing csc usage under daytime night time interactions

File: index.rst [hunk 1]
```diff
@@ -165,12 +165,4 @@ Contains detailed instructions on how to set the required environment for operat
 System Testing, Deployments, and Upgrades
 ==================================================================
```
Comment (patrickingraham) [inline index.rst:166]: You might get a warning since there are more = signs than characters in the title.

**Review by patrickingraham — APPROVED** [review]
OK with this. If you could make the two small changes from your other PR to the announcements page that'd be nice.

File: index.rst (hunk 1 above)
Comment (tribeiro) [reply index.rst:166]: yeah.. need to make a pass in the whole module.. but I don't want to make changes to the content when moving this much files around..

**Pull request merged**